Dockerfile

```diff
@@ -1,3 +1,11 @@
-FROM starefossen/github-pages:onbuild
+# Build the docs
+FROM starefossen/github-pages:147 AS build_docs
```

Comment:
One more thought: what if we created a `docs-builder` image, that is basically just:

```Dockerfile
FROM starefossen/github-pages:147
```

And change these Dockerfiles to:

```Dockerfile
FROM docs-builder:latest
```

Then, whenever we want to bump the version of Jekyll, we can just push that image, and all archive versions will use the new Jekyll version when rebuilding.

Using `:latest` may not be "best practice", but in this case would save a bit of maintenance :smile:

Reply:
We could stash the `nginx-override.conf` file in there as well, so that we can manage that one from a central location, and `COPY --from` that to the nginx image

Reply:
Or even: if we have other configuration files that we want to share between "master" and the archives (idk, perhaps there's some Jekyll configuration that needs to be kept in sync), we could have a `docs/docker.github.io:config` image that contains those files, then we can

```Dockerfile
FROM docs/docker.github.io:config AS configuration

...


COPY --from=docs-configuration /config/some-file /destination/
```